build/_k-standard-goals.shade

```diff
@@ -187,6 +189,18 @@ default Quiet='${ false }'
     }
   }
 
+#nuget-verify target='test' if='File.Exists(PACKAGELIST_JSON_FILENAME)' description='Verify if all the packages are generated properly'
+  exec program='cmd' commandline='/C dnu commands install --source ${DNX_TOOLS_FEED} NugetPackageVerifier --packages ${BASE_DIR}\commands\' if='ShouldVerifyNupkgs'
```

Comment:
Oh yeah there is, but does it work for command line apps that are installed as commands? Maybe it does...